interfaces/seccomp/backend.go

```diff
@@ -79,6 +79,16 @@ type Backend struct {
 	versionInfo seccomp.VersionInfo
 }
 
+// TODO: now that snap-seccomp has full support for deny-listing this
+// should be replaced with something like:
+//
+//	~ioctl - 4294967295|TIOCSTI
+//	~ioctl - 4294967295|TIOCLINUX
+//
+// in the default template. This requires that MaskedEq learns
+// to deal with two arguments (see also https://github.com/snapcore/snapd/compare/master...mvo5:rework-seccomp-denylist-incoperate-global.bin?expand=1)
+//
+// globalProfileLE is generated via cmd/snap-seccomp-blacklist
```

Comment:
Can you put a space to make the woke check happy before we address this properly in the follow-up?